QA/UNIT_TEST.md

```
@@ -0,0 +1,145 @@
+
+将来的には、GitHub Actions などを使用して以下を自動化する予定です：
+
+1. プルリクエスト時の自動テスト実行
+2. テストカバレッジレポートの生成と履歴管理
+3. テスト失敗時の自動通知
+
+## テストに関する注意事項
+
+- パスワード入力を要求するテストは `t.Skip()` でスキップされています
+- テスト実行時は一時ファイルが自動的に作成・削除されます
+- 環境変数 `ENVAULT_TEST_SKIP_INTERACTIVE=1` を設定すると対話式テストをスキップできます
```

markdownlint-cli2 (0.17.2)

Files should end with a single newline character
null

(MD047, single-trailing-newline)